### PR TITLE
[NOID] Snyk Hadoop updates

### PR DESCRIPTION
**Pull request by gem-neo4j**
Note: the comment to check javax.servlet.jsp:jsp-api:2.1 was done, and it hasn't updated so the comment stays for now

### CODE REVIEW

File: LICENSES.txt [hunk 1]
```diff
@@ -170,28 +170,29 @@ Apache-2.0
...
+  netty-handler-ssl-ocsp-4.1.89.Final.jar
   netty-resolver-4.1.93.Final.jar
-  netty-resolver-dns-4.1.77.Final.jar
-  netty-resolver-dns-classes-macos-4.1.77.Final.jar
-  netty-resolver-dns-native-macos-4.1.77.Final-osx-aarch_64.jar
-  netty-resolver-dns-native-macos-4.1.77.Final-osx-x86_64.jar
+  netty-resolver-dns-4.1.89.Final.jar
+  netty-resolver-dns-classes-macos-4.1.89.Final.jar
+  netty-resolver-dns-native-macos-4.1.89.Final-osx-aarch_64.jar
+  netty-resolver-dns-native-macos-4.1.89.Final-osx-x86_64.jar
```
Comment (Lojjs) [inline LICENSES.txt:195]: Probably there is a sensible reason but I cannot find these 2 dependencies when I list dependencies locally 🤷‍♀️

**Review by Lojjs — APPROVED** [review]
Tried to verify locally. There is 2 dependencies I cannot see but, I think it is probably ok as they have the same new version as the others

File: LICENSES.txt (hunk 1 above)
Comment (gem-neo4j) [reply LICENSES.txt:195]: I find: io.netty:netty-resolver-dns-native-macos:4.1.89.Final and am not sure exactly how it works with splitting out jars to -osx etc 🤔

Comment (Lojjs) [reply LICENSES.txt:195]: Same, I am not to worried about it though so please go ahead and merge

**Pull request merged**